frontend/javascripts/oxalis/model/accessors/dataset_accessor.ts

```diff
@@ -321,11 +323,11 @@ export function getDatasetExtentAsString(
 
   if (inVoxel) {
     const extentInVoxel = getDatasetExtentInVoxel(dataset);
-    return `${formatExtentWithLength(extentInVoxel, (x) => `${x}`)} voxel`;
+    return `${formatExtentInUnitWithLength(extentInVoxel, "vx", (x, _) => `${x}`)} voxel`;
```

Comment:
I think, the interface of formatExtentInUnitWithLength should be as before (the unit should not be passed separately). The unit can simply be accessed within the arrow function. In the current case, the unit is not needed (and formatExtentInUnitWithLength also only needs the unit to pass it to the lambda). Therefore, the code here wouldn't change at all, I think.
Maybe consider renaming the function.